dbt_subprojects/solana/models/_sector/staking/solana/staking_solana_stake_account_delegations.sql

```diff
@@ -0,0 +1,78 @@
+{{ config(
+    schema = 'staking_solana'
+    , alias = 'stake_account_delegations'
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['block_time', 'block_slot', 'stake_account', 'vote_account']
```

Comment:
looks like this model is failing the unique test, but only after incremental run. that usually means incorrect keys. i noticed this model has 4 total columns, 4 keys, meaning nothing to update when match found. can we simplify the keys to less columns?

then the other thing to consider is null values in a key, which would fail the lookup. can any be null?

Reply:
none of the keys can be full. and yes this would only ever append, and not update. I put this unique key here so that we just don't end up with duplicates for overlapping incremental runs. we can't remove any of them because stake accounts could move delegations between vote accounts (and vice versa) any time.

Reply:
okay, only other option should be the target filter. i've removed to try again.